### ISSUE TITLE
Icon may be slightly too big

### ISSUE THREAD

**Issue by tyirvine** [Gifox/Tracker#191]
<img width="279" alt="Screen Shot 2021-03-08 at 5 32 23 PM" src="https://user-images.githubusercontent.com/39813066/110400111-4ccba000-8034-11eb-99db-c72bd60955f5.png">

I'm not sure if this is due to Big Sur or not but the menu bar icon appears to be slightly oversized compared to other icons.

**Comment by iby** [comment]
Agree, it does feels a pixel too big. Will see what we can do in the next UI iteration.
